grc/core/platform.py

```diff
@@ -392,6 +393,19 @@ def get_output_language(self):
         return [(value, name, value == output_language_default)
                 for value, name in zip(param['options'], param['option_labels'])]
 
+    def _flowgraph_error_report(self, flowgraph):
```

Comment:
I like this change. For my edification, why make a separate method here? Why not just include all of this (appropriately) in the code above?

Reply:
@michaelld Originally, I intended to make this a method of the `Element` class, but there is no access to `Messages` there. I didn't place all of this in the exception because in the past I (and those that taught me) took a minimum-code-in-interrupts/exceptions approach, keeping exceptions easy to read at a glance. I can put it back in the exception if you like!

Reply:
I agree, this is good as a separate thing. It doesn't even need to be a method...

Reply:
@grant-cox thanks for the explanation; makes sense! I'm OK either way. This could be a function; it could be in Messages; it could be integrated as I mention; options! If we think there's any way this will be called from elsewhere, then making it a function makes total sense; if from some other Python script then I'd think moving it to Messages would make the most sense ... but, we might not know yet where else this might get called from, so leaving it as-is would also be OK ... we can always change things down the road if/as necessary.